src/components/gui/gui.jsx

```diff
@@ -6,8 +6,10 @@ const GreenFlag = require('../../containers/green-flag.jsx');
 const TargetPane = require('../../containers/target-pane.jsx');
 const Stage = require('../../containers/stage.jsx');
 const StopAll = require('../../containers/stop-all.jsx');
+const MenuBar = require('../menu-bar/menu-bar.jsx');
```

Comment:
We are introducing a mix of camelCase and hyphenated naming in this PR. Throughout the rest of 3.0 (with a few exceptions that we should clean) we try to stick with camelCase. @rschamp any thoughts on this?